code/index.html

```diff
@@ -1,37 +1,55 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="UTF-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <link rel="stylesheet" href="style.css" />
-    <title>Technigo Project Cluedo</title>
-  </head>
-  <body>
-    <section class="cards">
-      <article class="killer-deck" id="killerCard">
-        <p>KILLER</p>
-        <p id="killerName"></p>
-        <p id="killerAge"></p>
-        <p id="killerOccupation"></p>
-        <img id="killerImage" src="" height="100" />
-        <p id="killerDescription"></p>
-      </article>
-      <article class="weapon-deck" id="weaponCard">
-        <p>WEAPON</p>
-        <p id="weaponName"></p>
-        <p id="weaponWeight"></p>
-      </article>
-      <article class="room-deck" id="roomCard">
-        <p>ROOM</p>
-        <p id="roomName"></p>
-      </article>
-    </section>
-
-    <h1 id="mystery"></h1>
-
-    <button>REVEAL THE CRIME</button>
-
-    <script src="clue.js"></script>
-  </body>
-</html>
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+  <link rel="stylesheet" href="style.css" />
+
+  <link rel='icon' href='./assets/search.png' type='image/x-icon'/>
+  <title> Project Cluedo</title>
+</head>
+
+<body>
+  <h1>Cluedo</h1>
+  <h3>Do you have what it takes to solve a murder?</h3>
+  <h4>Click on the cards to find out!</h4>
+
+  <section class="cards">
+
+    <article class="killer-deck" id="killerCard">
+      <p>KILLER</p>
+      <p id="killerName"></p>
+      <p id="killerAge"></p>
+      <p id="killerOccupation"></p>
+      <img id="killerImage" src="" height="100" />
+      <p id="killerDescription"></p>
+      <p id="killerFavouriteWeapon"></p>
+    </article>
+
+    <article class="weapon-deck" id="weaponCard">
+      <p>WEAPON</p>
+      <p id="weaponName"></p>
+      <p id="weaponWeight"></p>
+      <p id="weaponMortality"></p>
+    </article>
+
+    <article class="room-deck" id="roomCard">
+      <p>ROOM</p>
+      <p id="roomName"></p>
+    </article>
+  </section>
+
+  <h1 id="mystery"></h1>
+
+  <button id="revealMystery">REVEAL THE CRIME</button>
+
+  <script src="clue.js"></script>
+
+  <footer>
+    <div> Sara Sundblom 2020</div>
```

Comment:
I love that you added a footer with your name in it, looks super professional ;) I need to start doing that!